src/chartCore/index.js

```diff
@@ -1,6 +1,8 @@
 // Copyright 2017 Kensho Technologies, LLC.
 
-exports.getForKey = require('./getForKey')
-exports.getForProps = require('./getForProps')
-exports.getLayers = require('./getLayers')
-exports.getScale = require('./getScale')
+import getForKey from './getForKey'
+import getForProps from './getForProps'
+import getLayers from './getLayers'
+import getScale from './getScale'
+
+export default {getForKey, getForProps, getLayers, getScale}
```

Comment:
Probably you want `export {default as getForKey} from './getForKey'`, etc. rather than a default export.

Otherwise, `export * as chartCore from './chartCore'` in `src/index.js` will not do what you expect in ESM (you will be exporting `{default: {getForKey, getForProps, getLayers, getScale}}` instead of `{getForKey, getForProps, getLayers, getScale}`

Reply:
Yes